tests/macros.rs

```diff
@@ -0,0 +1,66 @@
+// Mostly taken from
+// https://github.com/BurntSushi/ripgrep/blob/64ac2ebe0f2fe1c8967e7ec550bc32466bf40a07/tests/macros.rs
+// Licensed under the MIT License
+// Copyright (c) 2015 Andrew Gallant
+
+#[macro_export]
+macro_rules! rgtest {
```

Comment:
Could this just be replaced with https://github.com/colin-kiegel/rust-pretty-assertions?